src/functionalTest/java/uk/gov/hmcts/cft/idam/testingsupportapi/steps/BaseSteps.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.cft.idam.testingsupportapi.steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.config.EncoderConfig;
+import io.restassured.config.RestAssuredConfig;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import net.serenitybdd.rest.SerenityRest;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.cft.idam.testingsupportapi.config.EnvConfig;
+
+import java.nio.charset.StandardCharsets;
+
+public abstract class BaseSteps {
+
+    private static final RequestSpecification defaultSpec = requestSpecification(EnvConfig.TESTING_SUPPORT_API_URL);
+
+    private static String testingServiceClientToken;
+
+    private static RequestSpecification requestSpecification(String baseUri) {
+        final RestAssuredConfig config = RestAssuredConfig.newConfig()
+            .encoderConfig(EncoderConfig.encoderConfig().defaultContentCharset(StandardCharsets.UTF_8));
+        final RequestSpecBuilder specBuilder = new RequestSpecBuilder().setConfig(config).setBaseUri(baseUri)
+            .setRelaxedHTTPSValidation();
```

Comment:
We shouldn't need that anymore - all environments now have valid SSL certificates.